jaxrs/src/main/java/org/jboss/aerogear/unifiedpush/rest/metrics/PushMetricsEndpoint.java

```diff
@@ -85,11 +89,14 @@ public Response pushMessageInformationPerVariant(
             return Response.status(Response.Status.NOT_FOUND).entity("Could not find requested information").build();
         }
 
-        PageResult<PushMessageInformation> pageResult =
+        PageResult<PushMessageInformation, MessageMetrics> pageResult =
                 metricsService.findAllForVariant(id, extractSortingQueryParamValue(sorting), page, pageSize);
```

Comment:
This code, that is around the "findAllForVariant" is not executed w/ the new UI.

See: https://github.com/matzew/aerogear-unifiedpush-server/commit/3d3e682740a20d6c11a00db0b20ec8bca244a2bb
and: https://github.com/matzew/aerogear-unifiedpush-server/commit/0a8e3b56307dce5680cf209da5164c1e9d76948c

I am fine w/ doing a clean up AFTER this.   Or, @lfryc feel free to do on this PR as well